tests/rados/test_pg_split.py

```diff
@@ -0,0 +1,87 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from tests.rados.rados_test_util import create_pools, write_to_pools
+from tests.rados.stretch_cluster import wait_for_clean_pg_sets
+from utility.log import Log
+from utility.utils import method_should_succeed, should_not_be_empty
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Automates PG split and merge test scenarios.
+    1. Create replicated pool
+    2.
```

Comment:
Update the steps. This can be done as part of module docstring as well

Reply:
done